Tests/ValetIntegrationTests/CloudIntegrationTests.swift

```diff
@@ -58,41 +54,47 @@ class CloudIntegrationTests: XCTestCase
 
         // Setting
-        XCTAssertTrue(iCloudValet.setString("butts", forKey: "cloud"))
-        XCTAssertEqual("butts", iCloudValet.string(forKey: "cloud"))
-        XCTAssertNil(vanillaValet.string(forKey: "cloud"))
+        try iCloudValet.setString("butts", forKey: "cloud")
+        XCTAssertEqual("butts", try iCloudValet.string(forKey: "cloud"))
+        XCTAssertThrowsError(try vanillaValet.string(forKey: "cloud")) { error in
+            XCTAssertEqual(error as? KeychainError, .itemNotFound)
+        }
         
         // Removal
-        XCTAssertTrue(vanillaValet.setString("snake people", forKey: "millennials"))
-        XCTAssertTrue(iCloudValet.removeObject(forKey: "millennials"))
-        XCTAssertEqual("snake people", vanillaValet.string(forKey: "millennials"))
+        try vanillaValet.setString("snake people", forKey: "millennials")
+        try iCloudValet.removeObject(forKey: "millennials")
+        XCTAssertEqual("snake people", try vanillaValet.string(forKey: "millennials"))
     }
     
-    func test_setStringForKey()
+    func test_setStringForKey() throws
     {
-        allPermutations.forEach { valet in
-            XCTAssertNil(valet.string(forKey: key), "\(valet) read item from keychain that should not exist")
-            XCTAssertTrue(valet.setString(passcode, forKey: key), "\(valet) could not set item in keychain")
-            XCTAssertEqual(passcode, valet.string(forKey: key))
+        try allPermutations.forEach { valet in
+            XCTAssertThrowsError(try valet.string(forKey: key)) { error in
+                XCTAssertEqual(error as? KeychainError, .itemNotFound)
+            }
+            try valet.setString(passcode, forKey: key)
+            XCTAssertEqual(passcode, try valet.string(forKey: key))
         }
     }
     
-    func test_removeObjectForKey()
+    func test_removeObjectForKey() throws
     {
-        allPermutations.forEach { valet in
-            XCTAssertTrue(valet.setString(passcode, forKey: key), "\(valet) could not set item in keychain")
-            XCTAssertEqual(passcode, valet.string(forKey: key), "\(valet) read incorrect value from keychain.")
+       try allPermutations.forEach { valet in
+            try valet.setString(passcode, forKey: key)
+            XCTAssertEqual(passcode, try valet.string(forKey: key))
 
-            XCTAssertTrue(valet.removeObject(forKey: key), "\(valet) did not remove item from keychain.")
-            XCTAssertNil(valet.string(forKey: key), "\(valet) found removed item in keychain.")
+            try valet.removeObject(forKey: key)
+            XCTAssertThrowsError(try valet.string(forKey: key)) { error in
+                XCTAssertEqual(error as? KeychainError, .itemNotFound)
+            }
         }
     }
     
     // MARK: canAccessKeychain
     
     func test_canAccessKeychain()
     {
-        allPermutations.forEach { valet in
+       allPermutations.forEach { valet in
```

Comment:
nit: whitespace... this was correct before

Reply:
```suggestion
        allPermutations.forEach { valet in
```